.prow.yaml

```
@@ -19,6 +19,28 @@ presubmits:
               memory: "1Gi"
               cpu: "500m"
 
+
+  - name: thoth-mypy-py38
+    decorate: true
+    skip_report: false
+    always_run: true
+    context: aicoe-ci/prow/mypy
+    spec:
+      containers:
+        - image: quay.io/thoth-station/thoth-pytest-ubi8-py38:v0.12.9
+          command:
+            - "/usr/local/bin/mypy"
+            - "."
+            - "--ignore-missing-imports"
```

```suggestion
            - "--ignore-missing-imports"
            - "--exclude examples/*"
```

This will exclude the examples directory from mypy check